### PR TITLE
:recycle: More refactoring

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/__init__.py [hunk 1]
```diff
@@ -148,7 +151,8 @@ async def fetch_guilds(
         if after:
             params["after"] = after
 
-        data = await (await self._http.get("/users/@me/guilds", params=params)).json()
+        guilds = await self._http.get("/users/@me/guilds", params=params)
+        data = await guilds.json()
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:155]: Reverse the names?

**Review by TheUntraceable — APPROVED** [review]

**Comment by TheUntraceable** [comment]
Waiting for someone else to approve...

File: EpikCord/__init__.py (hunk 1 above)
Comment (Sigmanificient) [reply EpikCord/__init__.py:155]: That line was too long and the double await ain't good for typing, so i believe it's would be better to let a intermediate variable here

Comment (TheUntraceable) [reply EpikCord/__init__.py:155]: No I mean the names, guilds isn't the requests data. so `data = await request`, `guilds = await data.json()`

File: EpikCord/__init__.py [hunk 2]
```diff
@@ -53,6 +53,7 @@
 from .voice import *
 from .webhooks import *
 
+
```
Comment (TheUntraceable) [inline EpikCord/__init__.py:56]: What a contribution! Why didn't I think of this myself?!

**Review by TheUntraceable — APPROVED** [review]

**Pull request merged**